extensions/Worker.Extensions.EventGrid/src/EventGridTriggerAttribute.cs

```diff
@@ -1,11 +1,30 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-﻿using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+using Microsoft.Azure.Functions.Worker.Converters;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+using Microsoft.Azure.Functions.Worker.Extensions.EventGrid.TypeConverters;
 
 namespace Microsoft.Azure.Functions.Worker
 {
+    [AllowConverterFallback(true)]
+    [InputConverter(typeof(EventGridCloudEventConverter))]
+    [InputConverter(typeof(EventGridEventConverter))]
+    [InputConverter(typeof(EventGridBinaryDataConverter))]
+    [InputConverter(typeof(EventGridStringArrayConverter))]
```

Comment:
I am not seeing `[SupportsDeferredBinding]` attribute on any of these converters. Is this the expected behavior?

Reply:
Discussed offline

Reply:
Yeah we are not using deferred binding for this extension - it is not required as we already get all the information we need to do this without having to use ParameterBindingData